packages/integration-tests/cmr.js

```diff
@@ -325,7 +332,7 @@ async function getOnlineResourcesUMMG(cmrLink) {
  */
 async function getOnlineResources({ cmrMetadataFormat, cmrConceptId, cmrLink }) {
   if (cmrMetadataFormat === 'echo10') {
-    return getOnlineResourcesECHO10(cmrLink);
+    return getOnlineResourcesECHO10(cmrLink.replace(/(.echo10)$/, '.json'));
```

Comment:
I guess this is just an integration tests helper, but something about this feels a little dangerous. Do we not have unit tests for this function?

Reply:
No, generally the integration tests helpers don't have units.   A situation I'm not entirely happy with to be sure. 

That said, this is a fairly difficult regexp to screw up.

Reply:
Does this comment require followup in your view?

Reply:
Nah, it's fine. YOLO

Reply:
:shipit: